deploy/examples/ceph-client.yaml

```diff
@@ -18,3 +22,7 @@ spec:
   caps:
     mon: 'profile rbd'
     osd: 'profile rbd pool=volumes, profile rbd pool=vms, profile rbd-read-only pool=images'
+  # security:
+  #   cephx:
+  #     keyRotationPolicy: "KeyGeneration"
+  #     keyGeneration: 3
```

Comment:
I'm debating whether we want to keep these comments in the examples, but if we keep them, at least let's use a consistent `keyGeneration`, instead of this one `3` and above it's `2`.